lib/saml_idp/metadata_builder.rb

```diff
@@ -37,6 +37,14 @@ def fresh
                 Location: single_service_post_location
               descriptor.SingleSignOnService Binding: "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect",
                 Location: single_service_post_location
+              descriptor.SingleLogoutService Binding: "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST",
+                Location: single_logout_service_post_location
+              descriptor.SingleLogoutService Binding: "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect",
+                Location: single_logout_service_post_location
+              unless remote_logout_service_post_location.nil?
+                descriptor.SingleLogoutService Binding: "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST",
+                  Location: remote_logout_service_post_location
+              end
```

Comment:
I think positive branches are easier to read:
```suggestion
              if remote_logout_service_post_location.present?
                descriptor.SingleLogoutService Binding: "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST",
                  Location: remote_logout_service_post_location
              end
```

Reply:
Good call, for some reason in my head `#present?` wasn't available since it's not a Rails app, but it's part of a gem so that was silly. Thanks!